### PR TITLE
Modify ui

### PR DESCRIPTION
**Pull request by yusufsheiqh**

### CODE REVIEW

File: lib/make_move.rb [hunk 1]
```diff
@@ -5,16 +5,27 @@ def initialize(game_state_gateway:)
...
+    else
+      raise OccupiedError, 'Cannot make the same move twice'
     end
-    @game_state_gateway.save(new_board)
+    @game_state_gateway.save(board)
     true
   end
 end
+
+def check_mark_within_bounds(tile, board)
```
Comment (craigjbass) [inline lib/make_move.rb]: This method is defined outside of the class

File: lib/make_move.rb [hunk 2]
```diff
@@ -5,16 +5,27 @@ def initialize(game_state_gateway:)
...
+  raise IndexError, 'Tile out of bounds' unless tile.between?(
+    1, board.grid.length
+  )
+end
+
+class OccupiedError < StandardError
```
Comment (craigjbass) [inline lib/make_move.rb]: I'd avoid using exceptions to represent validation errors. Is it `exceptional` that players would attempt to cheat?

File: lib/make_move.rb (hunk 1 above)
Comment (yusufsheiqh) [reply lib/make_move.rb]: Thank you, will fix it now.

File: lib/make_move.rb (hunk 2 above)
Comment (dushan-madetech) [reply lib/make_move.rb]: No, we'll replace this to stop raising exceptions

**Pull request merged**